### PR TITLE
Update to Working group page to recruit new members

### PR DESCRIPTION
**Pull request by calvin-lau-sig7**
To test on preview page and for content review.

### CODE REVIEW

**Comment by calvin-lau-gds** [comment]
There's only draft content in so far - looking to review and refine it a bit. 
Wanted to see how it looks on the preview page.

File: src/community/design-system-working-group/index.md.njk [hunk 1]
```diff
@@ -20,8 +20,14 @@ By including representatives from a mixture of disciplines and departments, the
...
-## How to join the working group
+## Join the working group
 
-The GOV.UK Design System team is looking into the best way of helping people to join the working group.
+We’re looking for new members to help make the working group more diverse and better represent the wider public sector.
 
-In the meantime if you are interested in joining the working group email the Design System team at govuk-design-system-support@digital.cabinet-office.gov.uk
+If you think you can expand our range of voices across ethnicity, age, gender, experience and discipline, we want to hear from you.
+
+You’ll help promote collaboration across government and the Digital Transformation Strategy. Although our members volunteer their time with us, many of them do so as part of their community or corporate objectives.
```
Comment (CharlotteDowns) [inline src/community/design-system-working-group/index.md.njk:29]: @calvin-lau-gds can we find a link to the Digital Transformation Strategy?

Comment (CharlotteDowns) [reply src/community/design-system-working-group/index.md.njk:29]: We could even frame it around solving common user problems across government.

File: src/community/design-system-working-group/index.md.njk [hunk 2]
```diff
@@ -20,8 +20,14 @@ By including representatives from a mixture of disciplines and departments, the
...
+
+Find out more about what’s involved by reading our [Working group principles & ways of working](https://docs.google.com/document/d/1f6jUvxs94LjZXvHxiJIitm6_daKy2eYaVNVlYxmz5OE/edit?usp=sharing) document.
+
+If you’re interested, [sign up and tell us a bit about yourself](#). The deadline is 5pm on 26 February 2021.
```
Comment (CharlotteDowns) [inline src/community/design-system-working-group/index.md.njk:33]: Will this link to the 'get in touch' page or something else?

Comment (calvin-lau-gds) [reply src/community/design-system-working-group/index.md.njk:33]: Yes, Imran's working on a sign-up form which will be linked from here.

Ideal user journey would be something like:
- User sees post on social media or email
- Clicks on link to this landing page
- Finds information they need about the working group
- Clicks through to sign-up form and applies

File: src/community/design-system-working-group/index.md.njk (hunk 1 above)
Comment (calvin-lau-gds) [reply src/community/design-system-working-group/index.md.njk:29]: Probably link here?
https://www.gov.uk/government/publications/government-transformation-strategy-2017-to-2020

I've added a suggested edit to this gDoc:
https://docs.google.com/document/d/1jHqqb4IyTgv35oWgZynQSSyKhAOxW25snqWr_hdvnZw/edit?usp=sharing

**Comment by calvin-lau-sig7** [comment]
@ImranH-GDS: I've added a new commit with updated copy based on team input. Have a look!

Comment trail is in Google doc: https://docs.google.com/document/d/1jHqqb4IyTgv35oWgZynQSSyKhAOxW25snqWr_hdvnZw/edit#

**Comment by ImranH-GDS** [comment]
@calvin-lau-sig7  it's looking good thank you. This is likely to go out on Friday now. So we probably need to change the date to Friday 5th March.

Our user researcher is looking at the privacy notice for the form. Hopefully I should have the link with you soon.

**Comment by calvin-lau-sig7** [comment]
@ImranH-GDS Thanks! Survey link and latest changes have been committed - have a look! 
Once the survey's ready and you submit a review to 'Approve', I'll be able to merge this (to publish to live).

**Review by ImranH-GDS — APPROVED** [review]

**Pull request merged**